sdks/python/setup.py

```diff
@@ -141,7 +141,7 @@ def get_version():
     # server, therefore list of allowed versions is very narrow.
     # See: https://github.com/uqfoundation/dill/issues/341.
     'dill>=0.3.1.1,<0.3.2',
-    'fastavro>=0.21.4,<0.24',
+    'fastavro>=1.0.0,<2',
```

Comment:
Do we need to increase the lower bound (is the change not compatible with earlier fastavro releases)?

Reply:
Not really. I'll restore the previous bound